src/mysql.erl

```diff
@@ -136,7 +138,8 @@
                    {query_timeout, timeout()} |
                    {found_rows, boolean()} |
                    {query_cache_time, non_neg_integer()} |
-                   {tcp_options, [gen_tcp:connect_option()]},
+                   {tcp_options, [gen_tcp:connect_option()] |
+                   {ssl, term()}},
```

Comment:
Curly brace nesting error. The ssl options should be the value of the tcp_options.

Apart from that, good!

Reply:
oh, ok, fixed it =^^=